src/templates/common/contentFooter/index.tsx

```diff
@@ -12,44 +12,11 @@ type ContentFooterProps = {
   responsiveLayout?: string
 }
 
-function formatDate(date: Date, format: 'display' | 'machine'): string {
```

Comment:
Replaced this (and the `parseDate` and `getDateParts`) with `date-fns`. This function didn't care about time or timezones, so the behavior may be a little different using `date-fns`.